rust-console/src/main.rs

```diff
@@ -1,3 +1,3 @@
 fn main() {
-    println!("Hello, Toradex!");
+    println!("Hello Torizon!");
```

Comment:
please rebase your branch on top of `dev` and also take a look on the slint templates, I think that we need to stadardize there too.